test/e2e/resize_nodes.go

```diff
@@ -184,6 +184,24 @@ func waitForPodsCreated(c *client.Client, ns, name string, replicas int) (*api.P
 	return nil, fmt.Errorf("controller %s: Gave up waiting for %d pods to come up", name, replicas)
 }
 
+func waitForPodOnNode(c *client.Client, ns, name, node string) (*api.Pod, error) {
```

Comment:
There was no such method in util, so, I've moved it there (and rewritten to use wait.Poll).